tasks/prepare-fbc-release/tests/test-prepare-fbc-release-fail-no-data.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-prepare-fbc-release-fail-no-data
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the prepare-fbc-release task with no data file and verify the task fails as expected
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/redhat-appstudio/release-service-utils:447ea0580a2cdd48b4091e1df86fab5c3f86d01c
+            script: |
+              #!/usr/bin/env sh
+              set -eux
+
+              cat > $(workspaces.data.path)/snapshot_spec.json << EOF
+              {
+                  "application": "foo-app",
+                  "artifacts": {},
+                  "components": [
+                      {
+                          "containerImage": "quay.io/hacbs-release-tests/test-ocp-version/test-fbc-component@sha256:f6e744662e342c1321deddb92469b55197002717a15f8c0b1bb2d9440aac2297",
```

Comment:
This works. But typically we would use a dummy string that would be more obvious that it's a testing value - and it's easier to see that it's the same value in several places. I'm not saying it has to be that way, but something to consider, for consistency with other tests if nothing else.

Reply:
I see it was only changed in the no-data-file test, but not in the happy path test. Was that intentional?

Reply:
@mmalina 
Hi Martin, Yes, When I change it to `registry.io/image0@sha256:0000` in other places, I start to see this error while running the test.
```
[run-task : prepare-fbc-release] Found 2 components
[run-task : prepare-fbc-release] Mock skopeo called with: inspect --raw docker://registry.io/image0@sha256:0000
[run-task : prepare-fbc-release] parse error: Invalid numeric literal at line 1, column 5
```
So I thought I would keep it like this. I hope it's OK to keep it like this in the test.